src/utils/media-query-props.js

```diff
@@ -48,21 +48,23 @@ export class MediaQueryProps {
     this.exprMap_ = {};
   }
 
+  /**
+   * @param {string} queryString
+   * @return {boolean} value
+   */
+  resolveMatchQuery(queryString) {
```

Comment:
edit: figured it out. `resolveMatchQuery` essentially is about a matchQuery to _look_ like a listQuery s.t. you can pass it to the same resolve_ fn. I wonder if it would be simpler if we had separate codepaths, but i'm fine with this overall. Can we add a comment explaining that here?

Reply:
Added the comment to explain the query construction.